android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -59,17 +59,64 @@ public object WorkerBinder {
     this.workerBinderListenerWeakRef = WeakReference<WorkerBinderListener>(workerBinderListener)
   }
 
+  /**
+   * Bind a [Worker] (ie. a manager or any other class that needs an interactor's lifecycle) to an
+   * interactor's lifecycle events
+   *
+   * IMPORTANT: Binding will happen on [RibDispatchers.Default] unless the Worker explicitly
+   * overrides [coroutineContext] to other than [EmptyCoroutineContext]
+   *
+   * @param interactor The interactor that provides the lifecycle.
+   * @return [WorkerUnbinder] to unbind [Worker]'s lifecycle.
+   */
+  @JvmStatic
+  public fun Worker.bindTo(
```

Comment:
+1 to extension function. 

But since we're providing those new APIs as public extension function, I'd just remove them from inside the `WorkerBinder` object.

Consider it a nit.

Reply:
good catch! will be moving this outside WorkerBinder object in a sec

Reply:
@psteiger Decided to move all existing extensions to its own file so we can call the new extension functions on legacy java code

Reply:
Please ignore this. Reverted back to current WorkerBinder